src/main/scala/scorex/core/network/DeliveryTracker.scala

```diff
@@ -12,8 +14,12 @@ import scala.util.{Failure, Try}
 
 
 /**
-  * This class tracks modifier ids which are expected from and delivered by other peers
-  * in order to ban or de-prioritize peers which are delivering what is not expected
+  * This class tracks modifier statuses.
+  * Applied modifiers are not kept in this class as soon as we can get this status from history or mempool
```

Comment:
How can you get applied modifier status from mempool?

Reply:
If transaction was added to mempool, it is applied to it